packages/core/src/lib/time-tracking/time-slot/commands/handlers/update-time-slot-minutes.handler.ts

```
@@ -7,26 +7,36 @@
+	 */
 	public async execute(command: UpdateTimeSlotMinutesCommand): Promise<TimeSlotMinute> {
 		const { input, id } = command;
-		let timeMinute = await this.typeOrmTimeSlotMinuteRepository.findOneBy({
-			id
-		});
+
+		let timeMinute = await this.typeOrmTimeSlotMinuteRepository.findOneBy({ id });
 
 		if (timeMinute) {
+			// Prevent changing the timeSlot relationship during update
 			delete input.timeSlotId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)